vms/platformvm/vm_regression_test.go

```diff
@@ -1674,312 +1777,6 @@ func TestSubnetValidatorBLSKeyDiffAfterExpiry(t *testing.T) {
 	}
 }
 
-func TestPrimaryNetworkValidatorPopulatedToEmptyBLSKeyDiff(t *testing.T) {
```

Comment:
We added these tests when we had bugs on the P-Chain when migrating to BLS. I don't think these cases are even possible to reach now since all validators register BLS keys + we wouldn't even perform a lookback this far back in the P-Chain anyways.